### PR TITLE
improved debugAnnotation

### PR DESCRIPTION
**Pull request by klodav**

### CODE REVIEW

File: grails-app/controllers/de/laser/ajax/AjaxController.groovy [hunk 1]
```diff
@@ -1124,7 +1126,7 @@ class AjaxController {
             AuditConfig.removeAllConfigs(property)
 
             property.getClass().findAllByInstanceOf(property).each{ prop ->
-                prop.delete(flush: true) //see ERMS-2049. Here, it is unavoidable because it affects the loading of orphaned properties - Hibernate tries to set up a list and encounters implicitely a SessionMismatch
+                prop.delete() //see ERMS-2049. Here, it is unavoidable because it affects the loading of orphaned properties - Hibernate tries to set up a list and encounters implicitely a SessionMismatch
```
Comment (croqueGrec09) [inline grails-app/controllers/de/laser/ajax/AjaxController.groovy:1129]: Does this work? The comment was not there without reason ...

**Review by croqueGrec09 — COMMENTED** [review]
one remark

File: grails-app/controllers/de/laser/ajax/AjaxController.groovy (hunk 1 above)
Comment (klodav) [reply grails-app/controllers/de/laser/ajax/AjaxController.groovy:1129]: works

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**